src/components/toolbars/content-toolbar.js

```diff
@@ -216,10 +228,11 @@ const ContentToolbar = (props) => {
         return toolbarItems;
     }, [
         allowsDelete,
-        allowsDownloadCases,
+        allowsDownloadExportCases,
         allowsMove,
-        selectedElements.length,
-        stashedElements.length,
+        handleDownloadCases,
+        selectedElements,
+        stashedElements,
```

Comment:
this change could cause troubles, is it intended ?

Reply:
Restored` stashedElements.lenght`